bitnami/cassandra/templates/headless-svc.yaml

```diff
@@ -25,4 +36,4 @@ spec:
       port: {{ .Values.service.thriftPort }}
```

Comment:
Missing `service.nodePorts.thriftPort`

Reply:
What do you mean? This is a headless service.

Reply:
I meant that `service.nodePorts.thriftPort` is not used anyweher in the yaml but it is in values.yaml